src/cbc_sdk/platform/alerts.py

```diff
@@ -198,36 +199,53 @@ def get_observations(self, timeout=0):
         obs = Observation.bulk_get_details(self._cb, alert_id=alert_id, timeout=timeout)
         return obs
 
+    def get_history(self, threat=False):
+        """
+        Get the actions taken on an Alert such as Notes added and workflow state changes.
+
+        Returns:
```

Comment:
Based on Amy's guidelines this should not have a Returns section considering that other methods in this file explain what they are returning in the description of the methods.

Reply:
I disagree as we need to add returns in some cases like these to clarify what is specifically being returned. I think returns can be optional but it's not required to be dropped